src/main/java/pages/ShippingPage.java

```diff
@@ -0,0 +1,31 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class ShippingPage {
+    protected WebDriver driver;
+    private By titlePageCheckout = By.name("Checkout");
+    private By address = By.className("shipping-address-items");
+    private By addressSelected = By.className("shipping-address-item selected-item");
+    private By detailsQuantity = By.className("details-qty");
+
+    public ShippingPage(WebDriver driver){
+        this.driver = driver;
+    }
+
+public String verifyTittlePage (){
+        return driver.findElement(titlePageCheckout).getText();
+}
+
+public String verifyAddress(){
```

Comment:
should be called `getAddress()` because that's what you're doing, when a "check" requires a comparison between several things